CONTRIBUTING.rst

```diff
@@ -1,29 +1,24 @@
 Developing
 ==========
 
-Install dependencies:
+This project uses `Tox <https://tox.readthedocs.io/en/latest/config.html>`_ to manage virtual environments.
 
-::
+To run the tests::
 
-    pip install treq[dev]
+    tox -e py38-twisted_latest
 
-Run Tests (unit & integration):
+Lint::
 
-::
-
-    trial treq
-
-Lint:
-
-::
-
-    pep8 treq
-    pyflakes treq
+    tox -e flake8
 
 Build docs::
 
     tox -e docs
```

Comment:
Minor commet

Maybe also hint about where to find the resulting docs

```suggestion
    tox -e docs
    firefox docs/html/index.html
```

and maybe build the docs into build/docs so that the result is outside of the source and we don't need an extra gitignore